scripts/remove-use-strict.js

```
@@ -0,0 +1,35 @@
+
+files.forEach((file) => {
+  let content = fs.readFileSync(file, "utf8")
+
+  // Remove "use strict"; or "use strict" from the beginning of the file
+  content = content.replace(/^["']use strict["'];?\s*\n?/, "")
+
+  fs.writeFileSync(file, content, "utf8")
+})
+
+// eslint-disable-next-line no-console
+console.log(`Removed "use strict" from ${files.length} files`)
```

ESLint

[error] Insert `⏎`

(prettier/prettier)